src/ast-utils/descriptors.ts

```diff
@@ -0,0 +1,93 @@
+import * as estree from 'estree';
+import * as jsdoc from './jsdoc';
+import * as dom5 from 'dom5';
```

Comment:
do we have an import ordering style? I'm used to alphabetical, but in ES6, what would be alphabetized? The module name/path?